soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -823,6 +823,15 @@ void CreateAllHints() {
         }
     }
 
+    // If we have Rainbow Bridge set to Greg, add a hint for where Greg is
+    if (Bridge.Is(RAINBOWBRIDGE_GREG)) {
+      for(auto loc : allLocations) {
+        if (Location(loc)->GetPlacedItemKey() == GREG_RUPEE) {
```

Comment:
Seems like this could be placed in the filter logic above for "alwaysHintLocations", so we aren't having to loop through every check again just for this one hint